From ba903b0c9597a410faa2d48d2c7d21bab94f707c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Mon, 21 Oct 2024 10:46:19 +0200
Subject: [PATCH] ui: set tag selector button menu direction to up

This fixes the problem that the popup is invisible when there is no space
below the application.
---
 src/ui/moretagsgroup.blp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ui/moretagsgroup.blp b/src/ui/moretagsgroup.blp
index 95cb6d7..6298c4d 100644
--- a/src/ui/moretagsgroup.blp
+++ b/src/ui/moretagsgroup.blp
@@ -17,6 +17,7 @@ template $EartagMoreTagsGroup: Box {
   $EartagTagSelectorButton tag_selector {
     label: _("Add tag");
     tag-selected => $add_row_from_selector();
+    direction: "up";
     halign: "center";
     margin-top: "6";
 
-- 
GitLab

